examples/uncomplected/cats.py

```diff
@@ -0,0 +1,40 @@
+import random
+
+
+class Cat(object):
+    def __init__(self, name, last_ate=None):
+        self.name = name
+        self.last_ate = last_ate
+
+    def feed(self, food):
+        self.last_ate = food
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, self.__class__) and
```

Comment:
I wonder if the `isinstance` is required? Python is all about duck-typing. Maybe if the other thing has both a `name` and a `last_ate` it should be considered a Cat?